### PR TITLE
clean up css styles

### PR DESCRIPTION
**Pull request by ydegtyar**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [x] Code style update (formatting)
- [x] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes
- Fixed toolbar icon buttons tooltip text selector
- Deleted unused css styles

Issue Number: #140

### CODE REVIEW

File: src/css/styles.css [hunk 1]
```diff
@@ -56,8 +56,7 @@
   border-radius: 0;
 }
 
-.villain--static .villain-canvas,
-.villain--static .villain-canvas--overlay {
```
Comment (btzr-io) [inline src/css/styles.css]: Are you sure this is not used ?

Comment (ydegtyar) [reply src/css/styles.css]: `.villain-canvas--overlay` is not used anywhere in code. I checked this four times :)

Comment (btzr-io) [reply src/css/styles.css]: Lol, sorry no idea why I left that then :stuck_out_tongue:

**Comment by btzr-io** [comment]
@ydegtyar Nice, thanks :v:

**Pull request merged**